### PR TITLE
[Improve][docs] Add spark sql docs to task spark

### PR DESCRIPTION
**Pull request by sq-q**
## Purpose of the pull request

Add spark sql instructions 

## Brief change log

Add information on how to use spark how to execute spark sql program.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Comment by SbloodyS** [comment]
Related pr: #9790

**Review by SbloodyS — DISMISSED** [review]
LGTM, do you have time to take a look at this PR? @zhongjiajie

File: docs/docs/zh/guide/task/spark.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 ## 综述
 
-Spark  任务类型，用于执行 Spark 程序。对于 Spark 节点，worker 会通过使用 spark 命令 `spark submit` 方式提交任务。更多详情查看 [spark-submit](https://spark.apache.org/docs/3.2.1/submitting-applications.html#launching-applications-with-spark-submit)。
+Spark  任务类型，用于执行 Spark 程序。对于 Spark 节点，worker 会通过使用 spark 命令：
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/task/spark.md]:
```suggestion
Spark 任务类型用于执行 Spark 应用。对于 Spark 节点，worker 支持两个不同类型的 spark 命令提交任务：
```

File: docs/docs/zh/guide/task/spark.md [hunk 2]
```diff
@@ -22,11 +26,13 @@ Spark  任务类型，用于执行 Spark 程序。对于 Spark 节点，worker 
 - 失败重试间隔：任务失败重新提交任务的时间间隔，以分为单位。
 - 延迟执行时间：任务延迟执行的时间，以分为单位。
 - 超时警告：勾选超时警告、超时失败，当任务超过“超时时长”后，会发送告警邮件并且任务执行失败。
-- 程序类型：支持 Java、Scala 和 Python 三种语言。
+- 程序类型：支持 Java、Scala、Python 和 Sql 四种语言。
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/task/spark.md]:
```suggestion
- 程序类型：支持 Java、Scala、Python 和 SQL 四种语言。
```

File: docs/docs/zh/guide/task/spark.md [hunk 3]
```diff
@@ -40,30 +46,44 @@ Spark  任务类型，用于执行 Spark 程序。对于 Spark 节点，worker 
 
 ## 任务样例
 
-### 执行 WordCount 程序
+### (1) spark submit
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/task/spark.md]:
```suggestion
### spark submit
```

File: docs/docs/zh/guide/task/spark.md [hunk 4]
```diff
@@ -40,30 +46,44 @@ Spark  任务类型，用于执行 Spark 程序。对于 Spark 节点，worker 
...
 ![resource_upload](/img/tasks/demo/upload_jar.png)
 
-#### 配置 Spark 节点
+##### 配置 Spark 节点
 
 根据上述参数说明，配置所需的内容即可。
 
 ![demo-spark-simple](/img/tasks/demo/spark_task02.png)
 
+### (2) spark sql
```
Comment (zhongjiajie) [inline docs/docs/zh/guide/task/spark.md]:
```suggestion
### spark sql
```

File: docs/docs/en/guide/task/spark.md [hunk 5]
```diff
@@ -2,7 +2,11 @@
 
 ## Overview
 
-Spark task type used to execute Spark program. For Spark nodes, the worker submits the task by using the spark command `spark submit`. See [spark-submit](https://spark.apache.org/docs/3.2.1/submitting-applications.html#launching-applications-with-spark-submit) for more details.
+Spark task type for executing Spark programs. For Spark nodes, the worker will do this by using the spark command:
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/spark.md]:
```suggestion
Spark task type for executing Spark application. When executing the Spark task, the worker will submits a job to the Spark cluster by following commands:
```

File: docs/docs/en/guide/task/spark.md [hunk 6]
```diff
@@ -21,11 +25,13 @@ Spark task type used to execute Spark program. For Spark nodes, the worker submi
 - **Failed retry interval**: The time interval (unit minute) for resubmitting the task after a failed task.
 - **Delayed execution time**: The time (unit minute) that a task delays in execution.
 - **Timeout alarm**: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
-- **Program type**: Supports Java, Scala and Python.
+- **Program type**: Supports Java, Scala, Python and Sql.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/spark.md]:
```suggestion
- **Program type**: Supports Java, Scala, Python and SQL.
```

File: docs/docs/en/guide/task/spark.md [hunk 7]
```diff
@@ -39,30 +45,42 @@ Spark task type used to execute Spark program. For Spark nodes, the worker submi
 
 ## Task Example
 
-### Execute the WordCount Program
+### (1) spark submit
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/spark.md]:
```suggestion
### spark submit
```

File: docs/docs/en/guide/task/spark.md [hunk 8]
```diff
@@ -39,30 +45,42 @@ Spark task type used to execute Spark program. For Spark nodes, the worker submi
...
 ![resource_upload](/img/tasks/demo/upload_jar.png)
 
-#### Configure Spark Nodes
+##### Configure Spark Nodes
 
 Configure the required content according to the parameter descriptions above.
 
 ![demo-spark-simple](/img/tasks/demo/spark_task02.png)
 
+### (2) spark sql
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/spark.md]:
```suggestion
### spark sql
```

File: docs/docs/en/guide/task/spark.md [hunk 9]
```diff
@@ -39,30 +45,42 @@ Spark task type used to execute Spark program. For Spark nodes, the worker submi
...
+#### Execute DDL and DML statements
+
+This case is to create a view table terms and write three rows of data and a table wc in parquet format and determine whether the table exists. The program type is SQL. Insert the data of the view table terms into the table wc in parquet format.
+
+![spark_sql](/img/tasks/demo/spark_sql.png)
+
 ## Notice
 
-JAVA and Scala only used for identification, there is no difference. If you use Python to develop Spark application, there is no class of the main function and the rest is the same.
+JAVA and Scala are only used for identification, and there is no difference. If it is Spark developed by Python, there is no class for the main function. Others are the same. JAVA, Scala and Python do not have SQL scripts.
```
Comment (zhongjiajie) [inline docs/docs/en/guide/task/spark.md]:
```suggestion
JAVA and Scala are only used for identification, and there is no difference when you use the Spark task. If your application is developed by Python, you could just ignore the parameter **Main Class** in the form. Parameter **SQL scripts** is only for SQL type and could be ignored in JAVA, Scala and Python.
```

Comment (zhongjiajie) [inline docs/docs/en/guide/task/spark.md]:
```suggestion
JAVA and Scala are only used for identification, and there is no difference when you use the Spark task. If your application is developed by Python, you could just ignore the parameter **Main Class** in the form. Parameter **SQL scripts** is only for SQL type and could be ignored in JAVA, Scala and Python.
```

**Review by zhongjiajie — COMMENTED** [review]
Thanks for contributing, I had add some comments as below, and could you please use the English version of the screenshot

File: docs/docs/en/guide/task/spark.md (hunk 9 above)
Comment (sq-q) [reply docs/docs/en/guide/task/spark.md]: OK, I'll revise it now.

**Review by zhongjiajie — CHANGES_REQUESTED** [review]
Hi @sq-q, docs LGTM, but please change screenshot from Chinese to English

**Comment by sq-q** [comment]
> Hi @sq-q, docs LGTM, but please change screenshot from Chinese to English

ok, i'll cut it again

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks @sq-q

**Comment by sq-q** [comment]
> LGTM, thanks @sq-q

Grazie

**Pull request merged**